### PR TITLE
Design tweakin

### PR DESCRIPTION
**Pull request by DarthHater**
Did a quick design review with the Sonatype team, some mild tweaks based on feedback!

New view of Vulnerabilities, visually spaced a bit better, same color we apply to vulnerability highlights the field titles now, visually following it, bolded the title of the package as well. Using green for "good" packages now, which I love because Mariah Carey Xmas songs are the best and so are Xmas colors.

Screen Shot 2020-02-10 at 1 58 40 PM

Tweaks to startup OSS Index experience, visually set the version apart, as well as the "results" header

Screen Shot 2020-02-10 at 1 58 54 PM

IQ scan with a non-policy breaking build:

Screen Shot 2020-02-10 at 2 00 01 PM

IQ scan with a policy breaking build:

Screen Shot 2020-02-10 at 2 01 41 PM

Thanks to Christine for the help!!!!

### CODE REVIEW

**Comment by ButterB0wl** [comment]
Otherwise tested it out with both iq and ossi and looks great,

Does the report url turn red if it comes back with a failure here: 
image

**Comment by DarthHater** [comment]
@ButterB0wl we did orange, since it was eye grabbing without being super eye bleeding.

**Comment by ButterB0wl** [comment]
image
Love it

File: src/Audit/AuditIQServer.ts [hunk 1]
```diff
@@ -16,32 +16,19 @@
...
 
 export class AuditIQServer {
   public auditThirdPartyResults(results: ReportStatus): boolean {
     if (results.isError) {
-      console.log();
-      console.group();
-      console.error(results.errorMessage);
-      console.groupEnd();
-      console.log();
+      visuallySeperateText(results.errorMessage);
```
Comment (allenhsieh) [inline src/Audit/AuditIQServer.ts]: just checking this will just then be a console.log instead of the console.error (as before)?

Comment (DarthHater) [reply src/Audit/AuditIQServer.ts]: It comes down to if we want stuff going to `stderr` or `stdout`. Extended to do so in: 9c7bdf4

**Pull request merged**